src/analytics/platforms/platforms.v6.service.js

```diff
@@ -15,26 +15,35 @@ angular.module("ovh-api-services").service("OvhApiAnalyticsPlatformsV6", functio
     var adpResource = $resource("/analytics/platforms/:serviceName", {
         serviceName: "@serviceName"
     }, {
-        query: { method: "GET", isArray: true, cache: queryCache },
+        query: {
+            url: "/analytics/platforms",
```

Comment:
Specify the `url` isn't needed in the `query` action.

Reply:
This is removed. Thanks :)

Reply:
In addition your commit message should be:

```sh
fix(analytics.platforms): use the right url across actions # for instance
```